### PR TITLE
Add Base Prom Routes

### PR DESCRIPTION
**Pull request by Georgyskr**
### Description

Add PROM<>BSC collateral-collateral route for USDC token with Polygon having synthetic LP
Add PROM<>BSC collateral-collateral route for USDT token with Polygon having synthetic LP
Add PROM<>BSC native-collateral route for PROM token with Polygon having synthetic LP

### Backward compatibility
Yes

### Testing
PROM Route (Collateral-Native). Prom is ERC20 on ETH and Mainnet networks but Address(0) on Prom Mainnet. Polygon is chosen as an "LP" side

Prom -> Polygon
 https://prom-blockscout.eu-north-2.gateway.fm/tx/0xedc133b9bc36e8179274ad4013cdb7536ed2b7c9adc5e87e8fa96eaa085f14fc https://polygonscan.com/tx/0xa2022e6c179bc394be378391089e1ed174591162155de40456e4d501428e5603 
 
 Bsc -> Prom https://bscscan.com/tx/0xe010be218a95b9586bf1941daa2dc5b41eacac9c2ee1abe9567ea17b02efff4c
  https://prom-blockscout.eu-north-2.gateway.fm/tx/0x7742ceb73dd25e7f3f59fd29dfb1c678d232c7eddf9849b897644f6e8baaf8ef 
 
 — Attempt2 
 Prom -> Polygon
  https://prom-blockscout.eu-north-2.gateway.fm/tx/0x6eaffa32fd2d16b7b9fa66e72f24aeeebc9230f63298f052e1f3ca0d04ff8439 https://polygonscan.com/tx/0x1a20d8da537fe0cbf41298799156c3a8766e2261d628e18dc533370fee4caa2b
    BSC -> Prom https://bscscan.com/tx/0x1b2f4b035fb07a2acab20882a5ac6d58544f00986d348c1cbc14762e855dc01d
     https://prom-blockscout.eu-north-2.gateway.fm/tx/0xebc7b42b8dcca64aa3e64b16f604574477934ea898961fe78f72100708a39c20

USDT Router:
 Prom -> Polygon
  https://prom-blockscout.eu-north-2.gateway.fm/tx/0x2c985276f79128a37fe480c2a86a8adeccea38add89d530da3c5369c1e2140a3 https://polygonscan.com/tx/0x8b38efd4358d42eb598014de2875bbef3acfe33c7399e78ace21cc05a17d19d1
  
    Bsc -> Prom https://bscscan.com/tx/0x0f3e3d4685fb60fba3d090b5f5fbdfc30956848978559de29ecf05f596a672f0 https://prom-blockscout.eu-north-2.gateway.fm/tx/0xbbf88941afc32b76d39bdd32b0fe7ea5bd2c4a12b1c81b8d49afc6a9a9205ca6
    
 Polygon -> Bsc https://polygonscan.com/tx/0x1834678e7ac761a7ee8a94a550d4ec908e35a288c225fae22e506212c66ad2f6 https://bscscan.com/tx/0x59b9338ec9f68e01743f882d6fe692bb5ad62178bec1c05be8ffae0af1363abe

USDC Router:
BSC -> Polygon https://bscscan.com/tx/0xc894d03f6a910b8aa4f819efacf1b6352a6ec947a0c8936aa048526280dd6155 https://polygonscan.com/tx/0xb4dfa95ca0ac3d3bdfea889f2f94bfe67394ffa320cf149289441fb8d39854b7  
PROM -> BSC
 https://prom-blockscout.eu-north-2.gateway.fm/tx/0xeb59814312b09b97b61a15a40b120128bd34267da5f9436f19e25fad5cc1fc77 https://bscscan.com/tx/0xfecd2f73ed30af8ae82e13c244607ffa5c7f1b2a87a9e0a1668812c58ba44dd5

Polygon -> BSC
https://polygonscan.com/tx/0x8c3728e93e7fa3e00677953d2b14627110cf0dd5b1a09512f54e2fa963c78277 https://prom-blockscout.eu-north-2.gateway.fm/tx/0xdae6ad8cbdd6cc715fbcd166cf255283b65216dc8fc94aa6a0f0877b9b35f977

### CODE REVIEW

File: deployments/warp_routes/USDT/bsc-polygon-prom-config.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    standard: EvmHypSynthetic
+    symbol: USDT
+  - addressOrDenom: "0xDf6C58ec668daB037373BC49215722c2b0664484"
+    chainName: prom
+    collateralAddressOrDenom: "0x7d7b730CBc9C98cB423a6F206840028036d1DA48"
+    connections:
+      - token: ethereum|bsc|0x74e19D0c1828be422dDCd367a6ADa8CA7003d1a9
+      - token: ethereum|polygon|0xd9765D5fE176493808202e63E6B73b523D6A1C3A
+    decimals: 18
+    logoURI: /deployments/warp_routes/USDT/logo.svg
```
[the remark this answers is not in the document]
Comment (Georgyskr) [reply deployments/warp_routes/USDT/bsc-polygon-prom-config.yaml:29]: added logoURI to each file in detailed conics

File: deployments/warp_routes/USDC/bsc-polygon-prom-config.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    standard: EvmHypSynthetic
+    symbol: USDC
+  - addressOrDenom: "0x424712Bec7c94a0F804c50F77B641A24F33A138e"
+    chainName: prom
+    collateralAddressOrDenom: "0xcd1D86a6fDb32e9525C40a1a91892F33A0617E13"
+    connections:
+      - token: ethereum|bsc|0x647eeD57dFDd335f9B6f2a9DbDFe09f166b1AdaC
+      - token: ethereum|polygon|0xFDf46770Da4F907c09aEd92299552DC9c7996d99
+    decimals: 18
+    logoURI: /deployments/warp_routes/USDC/logo.svg
```
[the remark this answers is not in the document]
Comment (Georgyskr) [reply deployments/warp_routes/USDC/bsc-polygon-prom-config.yaml:29]: added logoURI to each file in detailed conics

File: deployments/warp_routes/PROM/bsc-ethereum-polygon-config.yaml [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# yaml-language-server: $schema=../schema.json
+tokens:
```
[the remark this answers is not in the document]
Comment (Georgyskr) [reply deployments/warp_routes/PROM/bsc-ethereum-polygon-config.yaml:2]: added logoURI to each file in detailed conics

File: deployments/warp_routes/PROM/bsc-polygon-prom-config.yaml [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    symbol: PROM
+  - addressOrDenom: "0x23Ae469604E3c1b581ADabf07c035472FbdB5Ee1"
+    chainName: polygon
+    connections:
+      - token: ethereum|bsc|0x700b9322F6Cce0c8EFF4754bE9AcE88Ce6582821
+      - token: ethereum|prom|0xc1f4b896B94bd57dEfB7b6e630Bb91995b86CbAC
+    decimals: 18
+    name: Prometeus
+    standard: EvmHypSynthetic
+    symbol: PROM
```
[the remark this answers is not in the document]
Comment (Georgyskr) [reply deployments/warp_routes/PROM/bsc-polygon-prom-config.yaml]: added logoURI to each file in detailed conics

File: deployments/warp_routes/USDT/USDT.svg
[the remark this answers is not in the document]
Comment (Georgyskr) [reply deployments/warp_routes/USDT/USDT.svg:1]: added logoURI to each file in detailed conics

File: deployments/warp_routes/USDC/USDC.svg
[the remark this answers is not in the document]
Comment (Georgyskr) [reply deployments/warp_routes/USDC/USDC.svg:1]: File deleted

File: .changeset/prom-initial-warp-routes.md
[the remark this answers is not in the document]
Comment (Georgyskr) [reply .changeset/prom-initial-warp-routes.md:1]: Have used the autogenerated changeset command and removed an old file

**Review by paulbalaji — APPROVED** [review]

**Pull request merged**